### PR TITLE
Updated readme for windows commands for installation

### PR DESCRIPTION
**Pull request by jerrying123**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,7 +50,34 @@
...
 
+2. Start all services with docker-compose
+    ```bash
+    cd katiml
+    type nul > .env
+    docker compose up --build
+    ```
+3. If you're starting for the first time, run the schema migration
+    ```bash
+    cd services/ingestion/schemas/pgsql
```
Comment (pharnoux) [inline README.md:68]: not a windows expert but aren't path using `\` ?

**Review by pharnoux — COMMENTED** [review]
Added a comment

File: README.md (hunk 1 above)
Comment (jerrying123) [reply README.md:68]: yes, but it also handles / in the same way

**Review by pharnoux — APPROVED** [review]
OK. There is conflict though.

**Pull request closed without merging**